### DISCUSSION TITLE
Custom CSS

### DISCUSSION THREAD

**Discussion by Tweekism** [jannis-baum/Vivify#83] [Q&A] [answered]
With the custom css do you need a complete css or just the classes you want to override?

**Comment by jannis-baum** [comment, marked as answer]
Just what you want to override. The custom CSS is added behind the default like this:

https://github.com/jannis-baum/vivify/blob/717c06844d2da6b8c65313ef1ca3025824ce451c/src/routes/viewer.ts#L71-L76

Here `config.styles` will be the content of the file at the path that you set in your config.
